installer/reckoner.spec

```diff
@@ -0,0 +1,32 @@
+# -*- mode: python -*-
+
+block_cipher = None
+
+
+a = Analysis(['cli.py'],
+             pathex=['.', '../', '/home/nhuanca/dev/reactiveops/reckoner/installer'],
```

Comment:
Should this have your path in it?  I think maybe we don't even need this file.

Reply:
haha neato :D must be autogenerated